.circleci/config.yml

```diff
@@ -8,6 +8,10 @@ jobs: # a collection of steps
     steps: # a collection of executable commands
       - checkout # special step to check out source code to working directory
 
+      - run: # run tests
+          name: global-install
```

Comment:
the build tool should also run locally. you make the tool depend on global packages... not good.